sync/class.jetpack-sync-wp-replicastore.php

```diff
@@ -42,27 +42,27 @@ function full_sync_end( $checksum ) {
 	public function post_count( $status = null, $min_id = null, $max_id = null ) {
 		global $wpdb;
 
-		$where = "";
+		$where = '';
 
 		if ( $status ) {
 			$where = "post_status = '" . esc_sql( $status ) . "'";
 		} else {
-			$where = "1=1";
+			$where = '1=1';
 		}
 
-		if ( $min_id != null ) {
-			$where .= " AND ID >= " . intval( $min_id );
+		if ( null != $min_id ) {
+			$where .= ' AND ID >= ' . intval( $min_id );
 		}
 
-		if ( $max_id != null ) {
-			$where .= " AND ID <= " . intval( $max_id );
+		if ( null != $max_id ) {
+			$where .= ' AND ID <= ' . intval( $max_id );
 		}
 
 		return $wpdb->get_var( "SELECT COUNT(*) FROM $wpdb->posts WHERE $where" );
 	}
 
 	public function get_posts( $status = null, $min_id = null, $max_id = null ) {
-		$args = array( 'orderby' => 'ID', 'posts_per_page' => -1 );
+		$args = array( 'orderby' => 'ID', 'posts_per_page' => - 1 );
```

Comment:
Is it really the standard to have a space between the sign of an integer and the number itself?